src/components/Zulip.js

```diff
@@ -61,23 +61,23 @@ class Zulip extends Component {
                     <ul>
                       <li>
                         <b>#newcomers: </b>
-                        {zulipstat[0] ? zulipstat[0].newcomers_messages : 0}
+                        {zulipstat && zulipstat[0] && zulipstat[0].newcomers_message ? zulipstat[0].newcomers_message : 0}
```

Comment:
This line looks repeated for `zulipstat && zulipstat[0]`. Can you please remove this `zulipstat && zulipstat[0]` from each line and add `zulipstat && zulipstat[0]` in parent tag (`ul`)?
```suggestion
                        {zulipstat[0].newcomers_message ? zulipstat[0].newcomers_message : 0}
```

Reply:
@codesankalp check now changes done.